src/index.js

```diff
@@ -1,22 +1,21 @@
 const express = require('express');
-const morgan = require('morgan');
+const bodyParser = require('body-parser');
 const app = express();
 const path = require('path');
-const mongoose = require('mongoose');
-
+const methodOverride = require("method-override");
+//var User = require('./model/user');
 // Settings
 app.set('port', 4000);
 app.set('views', path.join(__dirname, 'views'));
 app.engine('html', require('ejs').renderFile);
 app.set('view engine', 'ejs');
 
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
What are you doing in this line?

Reply:
I just read that it is not necessary for the express version, I will delete it